sampleproject/examples/urls.py

```diff
@@ -1,9 +1,57 @@
+import importlib
+
 from django.urls import path
 
-from examples.pages.form import FormPage
-from examples.pages.tabs import TabsPage
+from django_components.component import Component
+
+from .utils import discover_example_modules
+from .views import ExamplesIndexPage
+
+
+# For each example in `docs/examples/*`, register a URL pattern that points to the example's view.
+# The example will be available at `http://localhost:8000/examples/<example_name>`.
+# The view is the first Component class that we find in example's `page.py` module.
+#
+# So if we have an example called `form`:
+# 1. We look for a module `examples.dynamic.form.page`,
+# 2. We find the first Component class in that module (in this case `FormPage`),
+# 3. We register a URL pattern that points to that view (in this case `http://localhost:8000/examples/form`).
+def get_example_urls():
```

Comment:
This auto-registers URLs on the dev server for previewing the examples, like `http://localhost:8000/examples/form`